packages/react-devtools-core/webpack.backend.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
       __IS_FIREFOX__: false,
       __IS_CHROME__: false,
       __IS_EDGE__: false,
+      __IS_NATIVE__: true,
```

Comment:
Does this not conflict with the idea that `react-devtools` is also usable in mobile browsers or Safari? Or is `react-devtools` not using the version of `react-devtools-core/backend` that was built with this config?

Reply:
`packages/react-devtools` is a frontend-only packange and is only using `standalone` from `packages/react-devtools-core`. This webpack config is for backend file.

Reply:
Isn't `react-devtools` importing from `react-devtools-core/backend` (https://github.com/facebook/react/blob/9cdf8a99edcfd94d7420835ea663edca04237527/packages/react-devtools/index.js#L10)? I guess this is safe now but isn't it possible that we start bundling RN specific code that breaks module evaluation?

Reply:
Oh wow, I think this is out of date and should be removed. This file should not be used, because by default `bin.js` is going to start an electron session with React DevTools frontend.

Reply:
I believe this change did break this part
https://github.com/facebook/react/blob/v19.0.0/packages/react-devtools/README.md#usage-with-react-dom

> If you install react-devtools as a project dependency, you may also replace the <script> suggested above with a JavaScript import (import 'react-devtools'). It is important that this import comes before any other imports in your app (especially before react-dom). Make sure to remove the import before deploying to production, as it carries a large DevTools client with it. If you use Webpack and have control over its configuration, you could alternatively add 'react-devtools' as the first item in the entry array of the development-only configuration, and then you wouldn’t need to deal either with <script> tags or import statements.

How it supposed to work now?

Reply:
> I believe this change did break this part https://github.com/facebook/react/blob/v19.0.0/packages/react-devtools/README.md#usage-with-react-dom
> 
> > If you install react-devtools as a project dependency, you may also replace the <script> suggested above with a JavaScript import (import 'react-devtools'). It is important that this import comes before any other imports in your app (especially before react-dom). Make sure to remove the import before deploying to production, as it carries a large DevTools client with it. If you use Webpack and have control over its configuration, you could alternatively add 'react-devtools' as the first item in the entry array of the development-only configuration, and then you wouldn’t need to deal either with <script> tags or import statements.
> 
> How it supposed to work now?

For React Native, consider using React Native DevTools (available from 0.76+), which has RDT built-in:
https://reactnative.dev/docs/react-native-devtools

Reply:
But I do not use React Native 🙂 

I'm using a good old webpack to build to browser and getting this error:
```
ERROR in app
Module not found: Error: Can't resolve 'react-devtools' in '/Users/sergunich/dev/work/tromzo/stereo/code/dashboard'

<skipping bunch of lines for gravity>

                existing directory /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools
                  using description file: /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools/package.json (relative path: .)
                    using path: /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools/index
                      using description file: /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools/package.json (relative path: ./index)
                        no extension
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools/index doesn't exist
                        .js
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/sergunich/dev/work/tromzo/stereo/code/dashboard/node_modules/react-devtools/index.js doesn't exist
                          
<a bit more of lines>

```